std_msgs/Header header
bw_yolo/DetectionObject[] objects
uint32 width  # source image width
uint32 height  # source image height
